ocaml/lambda/printlambda.ml

```diff
@@ -376,19 +386,19 @@ let primitive ppf = function
   | Pasrint -> fprintf ppf "asr"
   | Pintcomp(cmp) -> integer_comparison ppf cmp
   | Pcompare_ints -> fprintf ppf "compare_ints"
-  | Pcompare_floats -> fprintf ppf "compare_floats"
+  | Pcompare_floats bf -> fprintf ppf "compare_floats %s" (boxed_float_name bf)
   | Pcompare_bints bi -> fprintf ppf "compare_bints %s" (boxed_integer_name bi)
   | Poffsetint n -> fprintf ppf "%i+" n
   | Poffsetref n -> fprintf ppf "+:=%i"n
-  | Pintoffloat -> fprintf ppf "int_of_float"
-  | Pfloatofint m -> fprintf ppf "float_of_int%s" (alloc_kind m)
-  | Pnegfloat m -> fprintf ppf "~.%s" (alloc_kind m)
-  | Pabsfloat m -> fprintf ppf "abs.%s" (alloc_kind m)
-  | Paddfloat m -> fprintf ppf "+.%s" (alloc_kind m)
-  | Psubfloat m -> fprintf ppf "-.%s" (alloc_kind m)
-  | Pmulfloat m -> fprintf ppf "*.%s" (alloc_kind m)
-  | Pdivfloat m -> fprintf ppf "/.%s" (alloc_kind m)
-  | Pfloatcomp(cmp) -> float_comparison ppf cmp
+  | Pintoffloat Pfloat64 -> fprintf ppf "int_of_float"
+  | Pfloatofint (Pfloat64,m) -> fprintf ppf "float_of_int%s" (alloc_kind m)
+  | Pabsfloat (bf,m) -> print_boxed_float "absf" ppf bf m
+  | Pnegfloat (bf,m) -> print_boxed_float "negf" ppf bf m
+  | Paddfloat (bf,m) -> print_boxed_float "addf" ppf bf m
+  | Psubfloat (bf,m) -> print_boxed_float "subf" ppf bf m
+  | Pmulfloat (bf,m) -> print_boxed_float "mulf" ppf bf m
+  | Pdivfloat (bf,m) -> print_boxed_float "divf" ppf bf m
```

Comment:
I preferred these names because they're now prefixed with `Float.`, like the Int64/Int32/etc functions. Removed the trailing `f`s, and also the dots from `float_comparison`.